### DISCUSSION TITLE
"Failed to compile" in react

### DISCUSSION THREAD

**Discussion by OmerAdam34** [neuefische/questions#214] [Web] [answered]
Hi, 

sobald ich eine Video Datei in meinem Code einbauen möchte, bekomme ich folgende Meldung: 
<img width="955" alt="Bildschirmfoto 2022-10-05 um 13 25 21" src="https://user-images.githubusercontent.com/107583280/194050807-58d1e7ec-f011-4091-942c-00391bb2775b.png">
<img width="542" alt="Bildschirmfoto 2022-10-05 um 13 35 05" src="https://user-images.githubusercontent.com/107583280/194051242-a32fbae4-b044-41ef-af70-feb3c7649556.png">

Muss bei den "dependencies" noch irgendwas hinzugefügt werden oder soll ich hier lieber mit einem package arbeiten? 
Leider finde ich für das Problem seit stunden keine Lösung

**Comment by julialoeschel** [comment, marked as answer]
Hey, did you seee this post? 
https://stackoverflow.com/questions/63649879/nextjs-throws-error-when-i-add-a-background-video?rq=1

seems to be the same Error message - It looks like you need a loader. 
did you already try this?

**Reply by OmerAdam34** [reply]
ahhh nice, it works :) thanks Coach J.

**Reply by ThomasKirst** [reply]
A similar error happens when you try to import an mp3 file.
We had a discussion here about it:
–> https://github.com/orgs/neuefische/discussions/168

You could also just place your video file in the `public` folder and reference it from there:
```
<source src="/my-video.mp4" … />
```

EDIT: Ah, yes, the answer on stackoverflow says it all ;)
